packages/uxpin-merge-cli/shell_scripts/test_resources/Makefile

```diff
@@ -12,22 +12,14 @@ test-resources: ../../test/resources/repos/nordnet-ui-kit ../../test/resources/r
 	&& yarn install --cache-folder ./yarn-cache \
 	&& rm -rf ./yarn-cache
 
-../../test/resources/repos/polaris:
-	cd ../../test/resources/repos \
-	&& git clone https://github.com/uxpin-merge/polaris-react.git polaris \
-	&& cd polaris \
-	&& mkdir ./yarn-cache \
-	&& yarn install --cache-folder ./yarn-cache \
-	&& rm -rf ./yarn-cache
-
 ../../test/resources/repos/mineral-ui:
 	cd ../../test/resources/repos \
 	&& git clone git@github.com:UXPin/mineral-ui.git \
 	&& cd mineral-ui \
 	&& git remote add official https://github.com/mineral-ui/mineral-ui.git \
 	&& git checkout 6a18faefe3d8339b066c92f1a10b27872f15729b \
 	&& mkdir ./npm-cache \
-	&& npm install --cache=./npm-cache \
+	&& npm install --force --legacy-peer-dep --cache=./npm-cache \
```

Comment:
It seems to fix the error related to peerDependencies. 
Note: there is no lock file for the resources, which makes the process fragile.